ramenctl/ramenctl/resources/configmap.yaml

```diff
@@ -36,12 +36,19 @@ data:
       s3CompatibleEndpoint: $minio_url_dr1
       s3Region: us-west-1
       s3SecretRef:
-        name: s3secret
+        name: ramen-s3
         namespace: ramen-system
+      VeleroNamespaceSecretKeyRef:
+        key: cloud
+        name: cloud-credentials
     - s3ProfileName: minio-on-dr2
       s3Bucket: bucket
       s3CompatibleEndpoint: $minio_url_dr2
       s3Region: us-east-1
       s3SecretRef:
-        name: s3secret
+        name: ramen-s3
         namespace: ramen-system
+      VeleroNamespaceSecretKeyRef:
+        key: cloud
+        name: cloud-credentials
+    veleroNamespaceName: velero
```

Comment:
Thanks, The changes were copied from
https://github.com/RamenDR/ramen/blob/99f7ab6e5551c2cc1173b81ac956d5a1cdaf877c/hack/recipe_e2e/config/ramen_config_base.yaml#L39

Reply:
@tjanssen3 I wonder if the config map referenced above is used by `recipe_e2e`?  It seems a few of the field identifiers are wrong.  Since this is not a CRD the usual API server checking is missing...maybe the fields are simply ignored, which would be fine for the Velero namespace name which defaults to `velero`, but the secret name and key do not have defaults.